requests-for-comments/commnunity-2024.md

```diff
@@ -0,0 +1,47 @@
+Up to now, much of our thinking for the community section of the website has centered on creating a Hacker News-like experience where content drives the community. I think this is the wrong direction. This issue outlines why and an alternative vision of the community.
+
+## Why not the current direction?
+
+Content, both our own and good content curated _by us_ can still form an important part of our community, but:
+
+1. There are already numerous places to share content about building startups / engineering topics generally. Why should people post it to ours and not another, more established channel?  
+
+2. The attraction of our content comes from our strong POV and curation. Outsourcing curation to the community undermines a strength we already have.
```

Comment:
This feels like a solvable probably. You could, say, introduce HN style upvoting, so that higher quality ones make it to the top

Reply:
We already have introduced HN-style upvoting that no one is using atm.